images/ceph/Makefile

```diff
@@ -22,6 +22,13 @@ CEPH_VERSION = v16.2.7-20220216
 else
 CEPH_VERSION = v16.2.7-20220216
 endif
+
+ifeq ($(CEPH_QUINCY),true)
+CEPH_VERSION = v17.2.0-20220420
+endif
+
+
+
```

Comment:
This is really unnecessary if we just make the CEPH_VERSION var overrideably by changing `CEPH_VERSION = <something>` to `CEPH_VERSION ?= <default-value>`.